.gitignore

```diff
@@ -23,4 +23,6 @@ server/.jest
 # misc
 react-ui/npm-debug.log*
 react-ui/yarn-debug.log*
-react-ui/yarn-error.log*
\ No newline at end of file
+react-ui/yarn-error.log*
+react-ui/package.json
+react-ui/package.json
```

Comment:
Hi Kyle,
We want to keep the package.json in GitHub so Heroku can view and install necessary dependencies for the front end. I would recommend removing lines 27 and 28 unless they are needed for another reason.

Reply:
alright I have removed lines 27 and lines 28